boa/network.py

```diff
@@ -421,6 +422,24 @@ def _warn_no_tracer():
 
         return call_tracer
 
+    def suppress_debug_tt(self, new_value=True):
+        self._suppress_debug_tt = new_value
+
+    def debug_tt(self, tx_hash):
+        if self._tracer is None:
+            return None
+        try:
+            return self._rpc.fetch_uncached(
+                "debug_traceTransaction", [tx_hash, self._tracer]
+            )
+        except RPCError as e:
+            if self._suppress_debug_tt:
+                warnings.warn(f"Couldn't get a trace for {tx_hash}")
+                return None
+            raise e
```

Comment:
I still think it'd be nice to retry if users are getting "transaction doesn't exist"

Reply:
i agree we should probably have a retry loop, but it deserves some care, so let's do that in a follow up PR.